<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Test Report</title>
    <link href="assets/style.css" rel="stylesheet" type="text/css"/></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) { // eslint-disable-line no-redeclare
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function findAll(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sortColumn(elem) {
    toggleSortStates(elem);
    const colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    let key;
    if (elem.classList.contains('result')) {
        key = keyResult;
    } else if (elem.classList.contains('links')) {
        key = keyLink;
    } else {
        key = keyAlpha;
    }
    sortTable(elem, key(colIndex));
}

function showAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(showExtras);
}

function hideAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(hideExtras);
}

function showExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.remove('collapsed');
    expandcollapse.classList.remove('expander');
    expandcollapse.classList.add('collapser');
}

function hideExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.add('collapsed');
    expandcollapse.classList.remove('collapser');
    expandcollapse.classList.add('expander');
}

function showFilters() {
    const filterItems = document.getElementsByClassName('filter');
    for (let i = 0; i < filterItems.length; i++)
        filterItems[i].hidden = false;
}

function addCollapse() {
    // Add links for show/hide all
    const resulttable = find('table#results-table');
    const showhideall = document.createElement('p');
    showhideall.innerHTML = '<a href="javascript:showAllExtras()">Show all details</a> / ' +
                            '<a href="javascript:hideAllExtras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    findAll('.col-result').forEach(function(elem) {
        const collapsed = getQueryParameter('collapsed') || 'Passed';
        const extras = elem.parentNode.nextElementSibling;
        const expandcollapse = document.createElement('span');
        if (extras.classList.contains('collapsed')) {
            expandcollapse.classList.add('expander');
        } else if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add('collapsed');
            expandcollapse.classList.add('expander');
        } else {
            expandcollapse.classList.add('collapser');
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener('click', function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains('collapsed')) {
                showExtras(event.currentTarget);
            } else {
                hideExtras(event.currentTarget);
            }
        });
    });
}

function getQueryParameter(name) {
    const match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () { // eslint-disable-line no-unused-vars
    resetSortHeaders();

    addCollapse();

    showFilters();

    sortColumn(find('.initial-sort'));

    findAll('.sortable').forEach(function(elem) {
        elem.addEventListener('click',
            function() {
                sortColumn(elem);
            }, false);
    });
}

function sortTable(clicked, keyFunc) {
    const rows = findAll('.results-table-row');
    const reversed = !clicked.classList.contains('asc');
    const sortedRows = sort(rows, keyFunc, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    const thead = document.getElementById('results-table-head');
    document.getElementById('results-table').remove();
    const parent = document.createElement('table');
    parent.id = 'results-table';
    parent.appendChild(thead);
    sortedRows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName('BODY')[0].appendChild(parent);
}

function sort(items, keyFunc, reversed) {
    const sortArray = items.map(function(item, i) {
        return [keyFunc(item), i];
    });

    sortArray.sort(function(a, b) {
        const keyA = a[0];
        const keyB = b[0];

        if (keyA == keyB) return 0;

        if (reversed) {
            return keyA < keyB ? 1 : -1;
        } else {
            return keyA > keyB ? 1 : -1;
        }
    });

    return sortArray.map(function(item) {
        const index = item[1];
        return items[index];
    });
}

function keyAlpha(colIndex) {
    return function(elem) {
        return elem.childNodes[1].childNodes[colIndex].firstChild.data.toLowerCase();
    };
}

function keyLink(colIndex) {
    return function(elem) {
        const dataCell = elem.childNodes[1].childNodes[colIndex].firstChild;
        return dataCell == null ? '' : dataCell.innerText.toLowerCase();
    };
}

function keyResult(colIndex) {
    return function(elem) {
        const strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
            'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[colIndex].firstChild.data);
    };
}

function resetSortHeaders() {
    findAll('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    findAll('.sortable').forEach(function(elem) {
        const icon = document.createElement('div');
        icon.className = 'sort-icon';
        icon.textContent = 'vvv';
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove('desc', 'active');
        elem.classList.add('asc', 'inactive');
    });
}

function toggleSortStates(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        resetSortHeaders();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function isAllRowsHidden(value) {
    return value.hidden == false;
}

function filterTable(elem) { // eslint-disable-line no-unused-vars
    const outcomeAtt = 'data-test-result';
    const outcome = elem.getAttribute(outcomeAtt);
    const classOutcome = outcome + ' results-table-row';
    const outcomeRows = document.getElementsByClassName(classOutcome);

    for(let i = 0; i < outcomeRows.length; i++){
        outcomeRows[i].hidden = !elem.checked;
    }

    const rows = findAll('.results-table-row').filter(isAllRowsHidden);
    const allRowsHidden = rows.length == 0 ? true : false;
    const notFoundMessage = document.getElementById('not-found-message');
    notFoundMessage.hidden = !allRowsHidden;
}
</script>
    <h1>Yatra.com Test Report</h1>
    <p>Report generated on 20-Dec-2021 at 13:12:40 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v3.1.1</p>
    <h2>Environment</h2>
    <table id="environment">
      <tr>
        <td>Packages</td>
        <td>{"pluggy": "1.0.0", "py": "1.11.0", "pytest": "6.2.5"}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Windows-10-10.0.19043-SP0</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{"html": "3.1.1", "metadata": "1.11.0"}</td></tr>
      <tr>
        <td>Python</td>
        <td>3.9.9</td></tr></table>
    <h2>Summary</h2>
    <p>3 tests ran in 126.62 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="passed">0 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="failed">3 failed</span>, <input checked="true" class="filter" data-test-result="error" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="error">0 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable" col="duration">Duration</th>
          <th class="sortable links" col="links">Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">TestCases/Test_search_flight.py::TestSearchAndVerifyFilter::test_search_flight_using_1_stop_1___New_Delhi____New____New_York__JFK_____25_12_2021____1_Stop__</td>
          <td class="col-duration">60.13</td>
          <td class="col-links"><a class="url" href="http://www.yatra.com/" target="_blank">URL</a> </td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div><div>Additional HTML</div></div>
            <div class="log">self = &lt;TestCases.Test_search_flight.TestSearchAndVerifyFilter testMethod=test_search_flight_using_1_stop_1___New_Delhi____New____New_York__JFK_____25_12_2021____1_Stop__&gt;<br/>going_from = &#x27;New Delhi&#x27;, going_to = &#x27;New&#x27;, search_to = &#x27;New York (JFK)&#x27;, date = &#x27;25/12/2021&#x27;, stops = &#x27;1 Stop&#x27;<br/><br/>    @data(*Utils.read_data_from_excel(&quot;C:\\Users\\BS238\\PycharmProjects\\AutomationFramework\\TestData\\test_data.xlsx&quot;, &quot;Sheet&quot;))<br/>    @unpack<br/>    def test_search_flight_using_1_stop(self, going_from, going_to, search_to,  date, stops):<br/>        # Manage all object in one line<br/>        search_flight = self.launch_page.search_flight(going_from, going_to, search_to, date)<br/>        # Dynamic scroll the full page<br/>        self.launch_page.page_scroll()<br/>        # 6. select the filter-1 stop<br/>        search_flight.filter_flight_by_stop(stops)<br/>        # 7. verify the filtered results show flights having only 1 stop<br/>        all_stop1 = search_flight.get_search_flight_results()<br/>        self.log.info(all_stop1)<br/>&gt;       self.util.assert_list_item_text(all_stop1, stops)<br/><br/>TestCases\Test_search_flight.py:43: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>Utilities\Utils.py:18: in assert_list_item_text<br/>    self.assert_all()<br/>..\..\AppData\Roaming\Python\Python39\site-packages\softest\case.py:139: in assert_all<br/>    self.fail(&#x27;&#x27;.join(failure_output))<br/><span class="error">E   AssertionError: ++++ soft assert failure details follow below ++++</span><br/><span class="error">E   </span><br/><span class="error">E   ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++</span><br/><span class="error">E   The following 9 failures were found in &quot;wrapper&quot; (Utilities.Utils.Utils):</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 1 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [1/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 2 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [2/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 3 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [3/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 4 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [4/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 5 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [5/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 6 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [6/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 7 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [7/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 8 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [8/9] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 9 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [9/9] +-</span><br/> -----------------------------Captured stderr setup------------------------------ <br/>

====== WebDriver manager ======
Current google-chrome version is 96.0.4664
Get LATEST chromedriver version for 96.0.4664 google-chrome
Driver [C:\Users\BS238\.wdm\drivers\chromedriver\win32\96.0.4664.45\chromedriver.exe] found in cache
<br/> -------------------------------Captured log setup------------------------------- <br/>INFO     WDM:logger.py:26 

INFO     WDM:logger.py:26 ====== WebDriver manager ======
INFO     WDM:logger.py:26 Current google-chrome version is 96.0.4664
INFO     WDM:logger.py:26 Get LATEST chromedriver version for 96.0.4664 google-chrome
INFO     WDM:logger.py:26 Driver [C:\Users\BS238\.wdm\drivers\chromedriver\win32\96.0.4664.45\chromedriver.exe] found in cache<br/> ------------------------------Captured stdout call------------------------------ <br/>The text is: 1 Stop
Test Passed
The text is: 1 Stop
Test Passed
The text is: 
Test Failed
The text is: 1 Stop
Test Passed
The text is: 1 Stop
Test Passed
The text is: 1 Stop
Test Passed
The text is: 1 Stop
Test Passed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 1 Stop
Test Passed
The text is: 1 Stop
Test Passed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 1 Stop
Test Passed
The text is: 1 Stop
Test Passed
The text is: 
Test Failed
The text is: 
Test Failed
<br/> ------------------------------Captured stderr call------------------------------ <br/>2021-12-20 13:10:54,944 - LaunchPage - INFO - New Delhi (DEL)
Indira Gandhi International
2021-12-20 13:10:54,966 - LaunchPage - INFO - India
2021-12-20 13:10:54,988 - LaunchPage - INFO - New York (NYC)
All airports
2021-12-20 13:10:55,010 - LaunchPage - INFO - United States
2021-12-20 13:10:55,031 - LaunchPage - INFO - New York (JFK)
John F Kennedy
2021-12-20 13:11:31,488 - SearchFlightResults - WARNING - Selected filter with 1 stop
2021-12-20 13:11:33,515 - TestSearchAndVerifyFilter - INFO - [&lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;9ad1c413-b1c8-4cf6-9cc4-bc471d3782fe&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;c56ea892-076b-4365-bccb-0fbbe0a6b7dc&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;3c03c6f5-0803-4dd1-9dc5-fefd9c2f61f5&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;d0419b76-197a-4d43-a41d-bdd90f93bacb&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;c98937f8-a385-4332-bcd6-56a5172234e8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;a16eb70f-00d2-4962-b1d8-8cd71968c52e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;f3985010-fd15-4c17-a6d7-1c50426ee1c8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;e58a3082-15be-42f4-be2d-213852ee1fd3&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;c5143781-43b6-45c8-9413-982e6ea649bb&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;51d5362a-a5e1-4757-9860-74cb18fdaeeb&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;a2f63375-8954-4088-ad03-6d26901396b2&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;eb72bf5a-4c6d-495b-a908-b148df00d51d&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;faf816e2-9613-4b32-97ee-4b5bda5ca901&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;4086799a-a11c-409a-b15c-8eecbf791092&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;660777de-65ae-492d-b00d-152f55113f1c&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;017c6f94-8c40-4009-b2c8-afa0e55cb4f6&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;e8dfdf35-12f8-47c1-a066-34dfbad2bd92&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;7e78bfb8-70e2-46c6-b96b-5f1fe8b0df44&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;1ce73ba7-1cef-40e3-a934-dbc9ee5cf905&quot;)&gt;]
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     LaunchPage:Yatra_launch_page.py:74 New Delhi (DEL)
Indira Gandhi International
INFO     LaunchPage:Yatra_launch_page.py:74 India
INFO     LaunchPage:Yatra_launch_page.py:74 New York (NYC)
All airports
INFO     LaunchPage:Yatra_launch_page.py:74 United States
INFO     LaunchPage:Yatra_launch_page.py:74 New York (JFK)
John F Kennedy
WARNING  SearchFlightResults:Search_flight_results_page.py:38 Selected filter with 1 stop
INFO     TestSearchAndVerifyFilter:Test_search_flight.py:42 [&lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;9ad1c413-b1c8-4cf6-9cc4-bc471d3782fe&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;c56ea892-076b-4365-bccb-0fbbe0a6b7dc&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;3c03c6f5-0803-4dd1-9dc5-fefd9c2f61f5&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;d0419b76-197a-4d43-a41d-bdd90f93bacb&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;c98937f8-a385-4332-bcd6-56a5172234e8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;a16eb70f-00d2-4962-b1d8-8cd71968c52e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;f3985010-fd15-4c17-a6d7-1c50426ee1c8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;e58a3082-15be-42f4-be2d-213852ee1fd3&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;c5143781-43b6-45c8-9413-982e6ea649bb&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;51d5362a-a5e1-4757-9860-74cb18fdaeeb&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;a2f63375-8954-4088-ad03-6d26901396b2&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;eb72bf5a-4c6d-495b-a908-b148df00d51d&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;faf816e2-9613-4b32-97ee-4b5bda5ca901&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;4086799a-a11c-409a-b15c-8eecbf791092&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;660777de-65ae-492d-b00d-152f55113f1c&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;017c6f94-8c40-4009-b2c8-afa0e55cb4f6&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;e8dfdf35-12f8-47c1-a066-34dfbad2bd92&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;7e78bfb8-70e2-46c6-b96b-5f1fe8b0df44&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;63bc2937465054e78e2f64d8ca38220e&quot;, element=&quot;1ce73ba7-1cef-40e3-a934-dbc9ee5cf905&quot;)&gt;]<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">TestCases/Test_search_flight.py::TestSearchAndVerifyFilter::test_search_flight_using_1_stop_2___BOM____SIN____Singapore__SIN_____30_12_2021____Non_Stop__</td>
          <td class="col-duration">52.37</td>
          <td class="col-links"><a class="url" href="http://www.yatra.com/" target="_blank">URL</a> </td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div><div>Additional HTML</div></div>
            <div class="log">self = &lt;TestCases.Test_search_flight.TestSearchAndVerifyFilter testMethod=test_search_flight_using_1_stop_2___BOM____SIN____Singapore__SIN_____30_12_2021____Non_Stop__&gt;<br/>going_from = &#x27;BOM&#x27;, going_to = &#x27;SIN&#x27;, search_to = &#x27;Singapore (SIN)&#x27;, date = &#x27;30/12/2021&#x27;, stops = &#x27;Non Stop&#x27;<br/><br/>    @data(*Utils.read_data_from_excel(&quot;C:\\Users\\BS238\\PycharmProjects\\AutomationFramework\\TestData\\test_data.xlsx&quot;, &quot;Sheet&quot;))<br/>    @unpack<br/>    def test_search_flight_using_1_stop(self, going_from, going_to, search_to,  date, stops):<br/>        # Manage all object in one line<br/>        search_flight = self.launch_page.search_flight(going_from, going_to, search_to, date)<br/>        # Dynamic scroll the full page<br/>        self.launch_page.page_scroll()<br/>        # 6. select the filter-1 stop<br/>        search_flight.filter_flight_by_stop(stops)<br/>        # 7. verify the filtered results show flights having only 1 stop<br/>        all_stop1 = search_flight.get_search_flight_results()<br/>        self.log.info(all_stop1)<br/>&gt;       self.util.assert_list_item_text(all_stop1, stops)<br/><br/>TestCases\Test_search_flight.py:43: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>Utilities\Utils.py:18: in assert_list_item_text<br/>    self.assert_all()<br/>..\..\AppData\Roaming\Python\Python39\site-packages\softest\case.py:139: in assert_all<br/>    self.fail(&#x27;&#x27;.join(failure_output))<br/><span class="error">E   AssertionError: ++++ soft assert failure details follow below ++++</span><br/><span class="error">E   </span><br/><span class="error">E   ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++</span><br/><span class="error">E   The following 37 failures were found in &quot;wrapper&quot; (Utilities.Utils.Utils):</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 1 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [1/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 2 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [2/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 3 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [3/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 4 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [4/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 5 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [5/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 6 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [6/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 7 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [7/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 8 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [8/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 9 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [9/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 10 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [10/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 11 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [11/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 12 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [12/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 13 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [13/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 14 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [14/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 15 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [15/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 16 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [16/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 17 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [17/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 18 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [18/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 19 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [19/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 20 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [20/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 21 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [21/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 22 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [22/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 23 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [23/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 24 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [24/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 25 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [25/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 26 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [26/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 27 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [27/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 28 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [28/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 29 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [29/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 30 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [30/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 31 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [31/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 32 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [32/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 33 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [33/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 34 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [34/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 35 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [35/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 36 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [36/37] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 37 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\Utilities\Utils.py&quot;, line 13, in assert_list_item_text</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\BS238\PycharmProjects\AutomationFramework\TestCases\Test_search_flight.py&quot;, line 43, in test_search_flight_using_1_stop</span><br/><span class="error">E       self.util.assert_list_item_text(all_stop1, stops)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\BS238\AppData\Roaming\Python\Python39\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Program Files\Python39\lib\unittest\case.py&quot;, line 676, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;1 Stop&#x27; != &#x27;Non Stop&#x27;</span><br/><span class="error">E   - 1 Stop</span><br/><span class="error">E   ? ^</span><br/><span class="error">E   + Non Stop</span><br/><span class="error">E   ? ^^^</span><br/><span class="error">E   </span><br/><span class="error">E   </span><br/><span class="error">E   -+ [37/37] +-</span><br/> -----------------------------Captured stderr setup------------------------------ <br/>

====== WebDriver manager ======
Current google-chrome version is 96.0.4664
Get LATEST chromedriver version for 96.0.4664 google-chrome
Driver [C:\Users\BS238\.wdm\drivers\chromedriver\win32\96.0.4664.45\chromedriver.exe] found in cache
<br/> -------------------------------Captured log setup------------------------------- <br/>INFO     WDM:logger.py:26 

INFO     WDM:logger.py:26 ====== WebDriver manager ======
INFO     WDM:logger.py:26 Current google-chrome version is 96.0.4664
INFO     WDM:logger.py:26 Get LATEST chromedriver version for 96.0.4664 google-chrome
INFO     WDM:logger.py:26 Driver [C:\Users\BS238\.wdm\drivers\chromedriver\win32\96.0.4664.45\chromedriver.exe] found in cache<br/> ------------------------------Captured stdout call------------------------------ <br/>The text is: 1 Stop
Test Failed
The text is: 1 Stop
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 1 Stop
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 1 Stop
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 1 Stop
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 
Test Failed
The text is: 1 Stop
Test Failed
The text is: 1 Stop
Test Failed
<br/> ------------------------------Captured stderr call------------------------------ <br/>2021-12-20 13:11:54,875 - LaunchPage - INFO - Singapore (SIN)
Changi
2021-12-20 13:12:23,480 - SearchFlightResults - WARNING - Selected filter with 2 stop
2021-12-20 13:12:25,502 - TestSearchAndVerifyFilter - INFO - [&lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;38a0e292-d659-463c-b538-5d061fbfdfb8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;d38f98ea-72cc-4b88-90fc-6a58ef1df690&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;6f536366-02bb-4300-a1f8-61550fadef86&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;4a23be99-ecc2-445e-8d2b-71f24e4b980e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;415d377b-6ebc-443c-97e6-66cd7e3069e1&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;b4f39f2c-9a7c-40b2-965b-208c8ef47a7a&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ef6047d0-4951-435f-a769-1af2acd8ae6b&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;1c7fb1fb-bd2d-40f0-a916-ce788c377d07&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;5fc5b4b0-29d3-496b-b0de-d4340fbbe535&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ebff630c-71e0-4c15-8af7-2d00c0317cd6&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;3072ddee-89a3-4700-be54-1663d9b8a23d&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;9427e8db-41a3-4c21-a1e8-872fad3da478&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;97213d44-ad7e-4d77-83fe-2f4d7ee4bfa9&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;0b14861c-8945-4e08-a820-d972a31e4205&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;d49bffa7-486d-4762-91f5-91c00a2095fd&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;971096c4-8a0b-4846-a15c-0a52e6d8f047&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;bb2b5831-e6f9-4f43-9ca1-22cc193a5fd2&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;1df59e42-172c-43f8-ba90-b58e04b6b2d0&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ea3ea0f4-7845-4686-8640-2442060c2fa5&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;5c64d410-8589-4ed8-acf5-772f1b799a8e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;8be917a1-cb2f-4da0-91a7-9698ce97f1e8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;34a5b401-5d7e-4d23-8b3b-6ee5446e2713&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;a5004a22-a2b4-407f-ae3f-e61ce32eebe7&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;8be51d6f-550b-434d-a928-7bcf64beb3b3&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;bb18fc38-2557-4d21-aca9-a33585d93365&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;8435132b-a169-480b-b1d7-5b8ae61e4ff2&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;34827b89-9551-457f-8796-f157428d9b37&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;5182d281-1fea-4d47-ade9-8be1b16847f9&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;e8b5f043-69a4-4905-9ab3-d5cfa61d5f42&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;93407270-2364-409a-b610-4a1f376935bf&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ebb1c31e-0ce3-4838-99f2-1b4ce3b1f402&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;d52d035c-1fdc-4054-b5a3-67b74638466e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ae18d941-0871-4060-9412-801d27ab19c4&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;f9992d84-27d7-420c-9885-b1e79c320769&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;64a84ae9-6841-4396-aae7-26221a1a22ab&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;47a147ec-b93a-421a-9a9b-8c885bc6529e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;0d49105f-1830-4582-a2e2-5fc551719fe1&quot;)&gt;]
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     LaunchPage:Yatra_launch_page.py:74 Singapore (SIN)
Changi
WARNING  SearchFlightResults:Search_flight_results_page.py:46 Selected filter with 2 stop
INFO     TestSearchAndVerifyFilter:Test_search_flight.py:42 [&lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;38a0e292-d659-463c-b538-5d061fbfdfb8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;d38f98ea-72cc-4b88-90fc-6a58ef1df690&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;6f536366-02bb-4300-a1f8-61550fadef86&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;4a23be99-ecc2-445e-8d2b-71f24e4b980e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;415d377b-6ebc-443c-97e6-66cd7e3069e1&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;b4f39f2c-9a7c-40b2-965b-208c8ef47a7a&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ef6047d0-4951-435f-a769-1af2acd8ae6b&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;1c7fb1fb-bd2d-40f0-a916-ce788c377d07&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;5fc5b4b0-29d3-496b-b0de-d4340fbbe535&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ebff630c-71e0-4c15-8af7-2d00c0317cd6&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;3072ddee-89a3-4700-be54-1663d9b8a23d&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;9427e8db-41a3-4c21-a1e8-872fad3da478&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;97213d44-ad7e-4d77-83fe-2f4d7ee4bfa9&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;0b14861c-8945-4e08-a820-d972a31e4205&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;d49bffa7-486d-4762-91f5-91c00a2095fd&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;971096c4-8a0b-4846-a15c-0a52e6d8f047&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;bb2b5831-e6f9-4f43-9ca1-22cc193a5fd2&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;1df59e42-172c-43f8-ba90-b58e04b6b2d0&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ea3ea0f4-7845-4686-8640-2442060c2fa5&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;5c64d410-8589-4ed8-acf5-772f1b799a8e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;8be917a1-cb2f-4da0-91a7-9698ce97f1e8&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;34a5b401-5d7e-4d23-8b3b-6ee5446e2713&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;a5004a22-a2b4-407f-ae3f-e61ce32eebe7&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;8be51d6f-550b-434d-a928-7bcf64beb3b3&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;bb18fc38-2557-4d21-aca9-a33585d93365&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;8435132b-a169-480b-b1d7-5b8ae61e4ff2&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;34827b89-9551-457f-8796-f157428d9b37&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;5182d281-1fea-4d47-ade9-8be1b16847f9&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;e8b5f043-69a4-4905-9ab3-d5cfa61d5f42&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;93407270-2364-409a-b610-4a1f376935bf&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ebb1c31e-0ce3-4838-99f2-1b4ce3b1f402&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;d52d035c-1fdc-4054-b5a3-67b74638466e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;ae18d941-0871-4060-9412-801d27ab19c4&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;f9992d84-27d7-420c-9885-b1e79c320769&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;64a84ae9-6841-4396-aae7-26221a1a22ab&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;47a147ec-b93a-421a-9a9b-8c885bc6529e&quot;)&gt;, &lt;selenium.webdriver.remote.webelement.WebElement (session=&quot;d2f9381900b3c92fd42bbc46a613c4e2&quot;, element=&quot;0d49105f-1830-4582-a2e2-5fc551719fe1&quot;)&gt;]<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">TestCases/Test_search_flight.py::TestSearchAndVerifyFilter::test_search_flight_using_1_stop_3__None__None__None__None__None_</td>
          <td class="col-duration">11.43</td>
          <td class="col-links"><a class="url" href="http://www.yatra.com/" target="_blank">URL</a> </td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div><div>Additional HTML</div></div>
            <div class="log">self = &lt;TestCases.Test_search_flight.TestSearchAndVerifyFilter testMethod=test_search_flight_using_1_stop_3__None__None__None__None__None_&gt;<br/>going_from = None, going_to = None, search_to = None, date = None, stops = None<br/><br/>    @data(*Utils.read_data_from_excel(&quot;C:\\Users\\BS238\\PycharmProjects\\AutomationFramework\\TestData\\test_data.xlsx&quot;, &quot;Sheet&quot;))<br/>    @unpack<br/>    def test_search_flight_using_1_stop(self, going_from, going_to, search_to,  date, stops):<br/>        # Manage all object in one line<br/>&gt;       search_flight = self.launch_page.search_flight(going_from, going_to, search_to, date)<br/><br/>TestCases\Test_search_flight.py:35: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>Pages\Yatra_launch_page.py:139: in search_flight<br/>    self.enter_depart_from_location(depart_location)<br/>Pages\Yatra_launch_page.py:47: in enter_depart_from_location<br/>    self.get_depart_from_field().send_keys(departLocation)<br/>..\..\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\remote\webelement.py:540: in send_keys<br/>    {&#x27;text&#x27;: &quot;&quot;.join(keys_to_typing(value)),<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>value = (None,)<br/><br/>    def keys_to_typing(value: Iterable[AnyKey]) -&gt; List[str]:<br/>        &quot;&quot;&quot;Processes the values that will be typed in the element.&quot;&quot;&quot;<br/>        typing: List[str] = []<br/>        for val in value:<br/>            if isinstance(val, Keys):<br/>                typing.append(val)<br/>            elif isinstance(val, int) or isinstance(val, float):<br/>                val = str(val)<br/>                for i in range(len(val)):<br/>                    typing.append(val[i])<br/>            else:<br/>&gt;               for i in range(len(val)):<br/><span class="error">E               TypeError: object of type &#x27;NoneType&#x27; has no len()</span><br/><br/>..\..\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\common\utils.py:151: TypeError<br/> -----------------------------Captured stderr setup------------------------------ <br/>

====== WebDriver manager ======
Current google-chrome version is 96.0.4664
Get LATEST chromedriver version for 96.0.4664 google-chrome
Driver [C:\Users\BS238\.wdm\drivers\chromedriver\win32\96.0.4664.45\chromedriver.exe] found in cache
<br/> -------------------------------Captured log setup------------------------------- <br/>INFO     WDM:logger.py:26 

INFO     WDM:logger.py:26 ====== WebDriver manager ======
INFO     WDM:logger.py:26 Current google-chrome version is 96.0.4664
INFO     WDM:logger.py:26 Get LATEST chromedriver version for 96.0.4664 google-chrome
INFO     WDM:logger.py:26 Driver [C:\Users\BS238\.wdm\drivers\chromedriver\win32\96.0.4664.45\chromedriver.exe] found in cache<br/></div></td></tr></tbody></table></body></html>